models/DataObject/ClassDefinition/Data/Objectbricks.php

```diff
@@ -226,10 +226,10 @@ private function getDataForField(Objectbrick\Data\AbstractData $item, string $ke
             $editmodeValue = $fielddefinition->getDataForEditmode($fieldValue, $baseObject, $params);
 
             if ($fielddefinition->isEmpty($fieldValue) && !empty($parent)) {
-                $backup = DataObject::getGetInheritedValues();
-                DataObject::setGetInheritedValues(true);
-                $parentItem = $parent->{'get' . ucfirst($this->getName())}()->$getter();
-                DataObject::setGetInheritedValues($backup);
+                $parentItem = DataObject\Service::useInheritedValues(function() use ($parent, $getter) {
+                    return $parent->{'get' . ucfirst($this->getName())}()->$getter();
+                }, true);
```

Comment:
What do you think about this:

```suggestion
                $parentItem = DataObject\Service::useInheritedValues(
                    fn () => $parent->{'get' . ucfirst($this->getName())}()->$getter(),
                    true,
                );
```